### DISCUSSION TITLE
V 1.25.0-1: Remove the Zigbee2MQTT configuration from the add-on configuration

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#12014] [General]
Hi all,
strip on changelog from the new version:

> Existing installations should be able to safely remove the Zigbee2MQTT configuration from the add-on configuration as these values should have already been persisted to the configuration file.

How can i do that?

If i go to the configuration page and delete the whole code and press save i get the error:

<img width="798" alt="image" src="https://user-images.githubusercontent.com/97361636/161233959-971be569-543d-4273-bb8f-8968db79f139.png">

**Comment by Koenkk** [comment]
@[joshuaspence](https://github.com/joshuaspence) could you help with this?

**Comment by alexyoung97** [comment]
I am also perplexed by these change notes, it's not clear to me which configuration file should now contain what. After the update to 1.25.0-1 my zigbee2mqtt web UI wouldn't open (502 bad gateway) due to a failure to find the zigbee adapter. Rebooting HA didn't help. However unplugging and plugging in the zigbee adapter seems to have done the trick; the web UI is back now. Mentioning this in case it helps anyone else who found this discussion following the update.
